services/storage-users/pkg/command/trash_bin.go

```diff
@@ -53,3 +89,402 @@ func PurgeExpiredResources(cfg *config.Config) *cli.Command {
 		},
 	}
 }
+
+func listTrashBinItems(cfg *config.Config) *cli.Command {
+	return &cli.Command{
+		Name:      "list",
+		Usage:     "Print a list of all trash-bin items of a space.",
+		ArgsUsage: "['spaceID' required]",
+		Flags:     []cli.Flag{},
+		Before: func(c *cli.Context) error {
+			return configlog.ReturnFatal(parser.ParseConfig(cfg))
+		},
+		Action: func(c *cli.Context) error {
+			log := logging.Configure(cfg.Service.Name, cfg.Log)
+			tp, err := tracing.GetServiceTraceProvider(cfg.Tracing, cfg.Service.Name)
+			if err != nil {
+				return err
+			}
+			var spaceID string
+			if c.NArg() > 0 {
+				spaceID = c.Args().Get(0)
+			}
+			if spaceID == "" {
+				_ = cli.ShowSubcommandHelp(c)
+				return fmt.Errorf("spaceID is requered")
+			}
+			fmt.Printf("Getting trash-bin items for spaceID: '%s' ...\n", spaceID)
+
+			ref, err := storagespace.ParseReference(spaceID)
+			if err != nil {
+				return err
+			}
+			client, err := pool.GetGatewayServiceClient(cfg.RevaGatewayGRPCAddr)
+			if err != nil {
+				log.Error().Err(err).Msg("error selecting next gateway client")
+				return err
+			}
+			ctx, err := utils.GetServiceUserContext(cfg.ServiceAccount.ServiceAccountID, client, cfg.ServiceAccount.ServiceAccountSecret)
+			if err != nil {
+				log.Error().Err(err).Msg("could not impersonate")
+				return err
+			}
+
+			spanOpts := []trace.SpanStartOption{
+				trace.WithSpanKind(trace.SpanKindClient),
+				trace.WithAttributes(
+					attribute.KeyValue{Key: "spaceID", Value: attribute.StringValue(spaceID)},
+				),
+			}
+			ctx, span := tp.Tracer("storage-users trash-bin list").Start(ctx, "serve static asset", spanOpts...)
+			defer span.End()
+
+			res, err := client.ListRecycle(ctx, &provider.ListRecycleRequest{Ref: &ref, Key: "/"})
+			if err != nil {
+				log.Error().Err(err).Msg(retrievingErrorMsg)
+				return err
+			}
+			if res.Status.Code != rpc.Code_CODE_OK {
+				return fmt.Errorf("%s %s", retrievingErrorMsg, res.Status.Code)
+			}
+
+			if len(res.GetRecycleItems()) > 0 {
+				fmt.Println("The list of the trash-bin items. Use an itemID to restore.")
+			} else {
+				fmt.Println("The list is empty.")
+			}
+
+			for _, item := range res.GetRecycleItems() {
+				fmt.Printf("itemID: '%s', path: '%s', type: '%s',  delited at :%s\n", item.GetKey(), item.GetRef().GetPath(), itemType(item.GetType()), utils.TSToTime(item.GetDeletionTime()).UTC().Format(time.RFC3339))
```

Comment:
Would look nicer if we could use some cli table writer here. We already use `olekukonko/tablewriter` in some places. See here for example: https://github.com/owncloud/ocis/blob/master/services/idp/pkg/command/version.go#L38-L46 

What do you think?